src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -154,17 +156,20 @@ def _assess_clusters(self, all_clusters):
         for cluster in all_clusters:
             if cluster.cluster_source == ClusterSource.JOB:
                 continue
-            if not cluster.creator_user_name:
+            creator = cluster.creator_user_name
+            if not creator:
                 logger.warning(
                     f"Cluster {cluster.cluster_id} have Unknown creator, it means that the original creator "
                     f"has been deleted and should be re-created"
                 )
+                # Normalize empty creator.
```

Comment:
Normalize?

Reply:
The prior code had a code-path where `""` and `None` were both treated as missing, and really they're the same thing so normalising to `None` makes sense. It's been slightly simplified now.